source/common/types/stateDirectory.types.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export type CheckStateDirectoryResponse = {
+  stateDirectory: string,
```

Comment:
@DeeJayElly can you please use this name for the response: stateDirectoryPath? 👍